extra/1-currency-conversion.js

```diff
@@ -5,7 +5,11 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
+function convertToUSD(price) {
```

Comment:
I think you meant `pricePound` here. 
Otherwise it's  good idea to use the currency in the name of the variable or parameter. Same with units: `distanceInMeters`, `waveLengthCm`, etc. It can be useful to avoid any confusion when sharing your code.